Makefile

```diff
@@ -233,8 +233,12 @@ drone: $(DRONE)
 	$(DRONE) lint .drone.yml --trusted
 	$(DRONE) --server https://drone.grafana.net sign --save grafana/grafana
 
+# Generate an Emacs tags table (https://www.gnu.org/software/emacs/manual/html_node/emacs/Tags-Tables.html) for Starlark files.
+scripts/drone/TAGS: $(shell find scripts/drone -name '*.star')
+	etags --lang none --regex="/def \(\w+\)[^:]+:/\1/" --regex="/\s*\(\w+\) =/\1/" $^ -o $@
+
 format-drone:
-	black --include '\.star$$' -S scripts/drone/ .drone.star
```

Comment:
Wow, wasn't aware that black also somehow worked with starlark 🤯

Reply:
Haha, I wouldn't have thought of it either. I believe it was added because of Starlark's syntactic similarity to Python.